packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -218,11 +246,15 @@ export class Pagination
   };
 
   private showLeftEllipsis() {
-    return Math.floor(this.startItem / this.pageSize) > 3;
+    return Math.floor(this.startItem / this.pageSize) > this.maxPagesDisplayed - 2;
```

Comment:
Can we also store `2` in a self-documenting const? I believe the same value would be used in `showRightEllipsis`, right?